src/engine/SCons/Tool/MSCommon/vc.py

```diff
@@ -321,7 +323,30 @@ def is_host_target_supported(host_target, msvc_version):
     return True
 
 
-def find_vc_pdir_vswhere(msvc_version):
+VSWHERE_PATHS = [os.path.join(p,'vswhere.exe') for p in  [
+    os.path.expandvars(r"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer"),
+    os.path.expandvars(r"%ProgramFiles%\Microsoft Visual Studio\Installer"),
+    os.path.expandvars(r"%ChocolateyInstall%\bin"),
```

Comment:
I'm not sure how I feel about having Chocolatey in the default list of search locations. It feels a bit like overreach. If you want the one out of Chocalatey for some reason, now you can name it explicitly via Variables by using the configurable `VSWHERE`. Maybe this should be removed?

Reply:
It's been there for a while.  It seems a reasonably common method to install packages on windows.
In general if you have MSVC 2017 or above installed it won't find it there.
I think it's a reasonable default.

Reply:
Worth noting. SCons will not add the Chocolatey install PATH into PATH for the env.
This is only used for running vswhere.exe